### ISSUE TITLE
Add a new payload

### ISSUE THREAD

**Issue by buggii** [Switch-Hax-uf2/switch-.uf2-files#1]
New ArgonNX-SE released ! Can you please convert its .bin file to .uf2 ? 

Here it is the .bin file:
https://mega.nz/file/nhVkFDgI#BOzHxi5vmdcsfT9--I1bW0WRJkzHepd16ce3OZ2XIJo

Thank you !

**Comment by Switch-Hax-uf2** [comment]
I have added the file under /uf2 files. I used the payload from Storm21CH's archive (that is what comes up when I searched ArgonNX-SE) just to make sure it's the latest version. This converter was not intended for Mariko switches so use it at your own risk, but if it injects the same as the gen 1 switch it should work. Let me know how it turns out.
[Link to file](https://github.com/Switch-Hax-uf2/switch-.uf2-files/raw/master/uf2%20files/argon-nxsh.uf2)
